### PR TITLE
User reset password

### PR DESCRIPTION
**Pull request by SteRiccio**

### CODE REVIEW

File: core/i18n/resources/en.js [hunk 1]
```diff
@@ -141,13 +142,19 @@ export const enTranslation = {
...
     login: 'Login',
     forgotPassword: 'Forgot password',
     sendVerificationCode: 'Send verification code',
   },
 
+  resetPasswordView: {
+    setNewPassword: 'Set new password',
+    forgotPasswordLinkInvalid: 'Invalid or expired link',
+    passwordSuccessfullyReset: 'Your password has been successfully reset',
+    goToLoginPage: 'Go to login page',
```
Comment (minotogna) [inline core/i18n/resources/en.js]: where is this used?

File: core/i18n/resources/en.js [hunk 2]
```diff
@@ -727,19 +731,25 @@ Do you want to leave the page?`,
...
+             $t(emails.signature)`,
       temporaryPasswordMsg:
         '<p>Your username is <strong>{{email}}</strong> and your temporary password is <strong>{{password}}</strong></p>',
     },
+    userForgotPassword: {
+      subject: 'OpenForis Arena: password forgot',
+      body: `<p>Hello,</p>
+             <p>You have asked a new password to access the system.</p>
+             <p><a href="{{url}}">Click here to reset your password</a></p>
+             $t(emails.signature)`,
```
Comment (minotogna) [inline core/i18n/resources/en.js]: I would say:

```
Hello {{name}},

Someone requested that the password for your OpenForis Arena account be reset.

<a href="{{url}}">Click here to reset your password</a>

If you didn't request this, you can ignore this email or let us know. Your password won't change until you create a new password. 

{{signature}}
```

File: core/i18n/resources/en.js (hunk 1 above)
Comment (SteRiccio) [reply core/i18n/resources/en.js]: it's unused, I'll remove it

File: core/i18n/resources/en.js (hunk 2 above)
Comment (SteRiccio) [reply core/i18n/resources/en.js]: Also we can add `This link will expire in 48 hours.`

Comment (minotogna) [reply core/i18n/resources/en.js]: yes

**Pull request merged**